synapse/tools/autodoc.py

```diff
@@ -291,49 +303,156 @@
                          ''
                          )
 
+        props = [blob for blob in props if blob[0] not in univ_names]
         if props:
-            rst.addLines('Properties:',
+
+            has_opts = any((popts for _, _, popts in props))
+
+            widths = '10 10 60 20'
```

Comment:
Since there are a lot of fixed widths in here, did you consider a test that would verify the model fits within these so we can avoid wrapping behavior later on?

Reply:
changing width to auto avoids fixed percentages and lets the table be more responsive.